10.epilepsy_path_ranking.ipynb

```diff
@@ -329,99 +327,99 @@
-       "920789  0.188982    CtDrD             5.108510   \n",
-       "920790  0.188982    CtDrD             5.108510   \n",
-       "920788  0.188982    CtDrD             5.108510   \n",
+       "              PDP metapath  metapath_importance  \\\n",
+       "1181486  0.011800    CbGaD             1.968488   \n",
+       "1133753  0.000303  CbGdAlD             2.186818   \n",
+       "1133754  0.000155  CbGdAlD             2.186818   \n",
+       "1133755  0.000153  CbGdAlD             2.186818   \n",
+       "1410458  0.000846  CtDdGaD             1.106620   \n",
        "\n",
-       "                                                    path  path_importance  \\\n",
-       "943186  Ethinamate–Gabapentin–migraine–epilepsy syndrome       519.089564   \n",
-       "921512          Amitriptyline–migraine–epilepsy syndrome       511.367241   \n",
-       "920789             Gabapentin–migraine–epilepsy syndrome       510.850993   \n",
-       "920790            Propranolol–migraine–epilepsy syndrome       510.850993   \n",
-       "920788          Valproic Acid–migraine–epilepsy syndrome       510.850993   \n",
+       "                                                      path  path_importance  \\\n",
+       "1181486                     Abacavir–ADK–epilepsy syndrome       196.848818   \n",
+       "1133753      Abacavir–ADH6–telencephalon–epilepsy syndrome        68.693397   \n",
+       "1133754  Abacavir–ADH6–medulla oblongata–epilepsy syndrome        35.187079   \n",
+       "1133755         Abacavir–ADH6–cerebellum–epilepsy syndrome        34.732125   \n",
+       "1410458  Abacavir–acquired immunodeficiency syndrome–HS...        24.962412   \n",
        "\n",
-       "        percent_of_DWPC         source  \n",
-       "943186            100.0     Ethinamate  \n",
-       "921512            100.0  Amitriptyline  \n",
-       "920789            100.0     Gabapentin  \n",
-       "920790            100.0    Propranolol  \n",
-       "920788            100.0  Valproic Acid  "
+       "         percent_of_DWPC    source  \n",
+       "1181486       100.000000  Abacavir  \n",
+       "1133753        31.412493  Abacavir  \n",
+       "1133754        16.090540  Abacavir  \n",
+       "1133755        15.882496  Abacavir  \n",
+       "1410458        22.557351  Abacavir  "
       ]
      },
-     "execution_count": 9,
+     "execution_count": 25,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "result_df = result_df.sort_values(by=['path_importance'], ascending=False)\n",
+    "result_df = result_df.sort_values(by=['source', 'path_importance', 'metapath'], ascending=[True, False, True])\n",
     "result_df.head()"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "result_df.to_csv('all_paths.tsv.gz')"
+    "result_df.to_csv('all_paths.tsv.gz', index=False)"
```

Comment:
AHH, we must specify `sep='\t'` otherwise this is a CSV not a TSV, my pet peeve.